src/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Source files
+file(GLOB_RECURSE SOURCES_SC2LADDERSERVER "*.cpp" "*.h")
```

Comment:
Recurse is not needed here because you don't have embedded directories under src.

Reply:
Yes, this is true.
I set it to recurse so as to (hopefully) future proof it in case folders are created.